content/post/080-big-refactors.md

```diff
@@ -0,0 +1,26 @@
+---
+date: 2020-01-31
```

Comment:
Maybe delay this by a day or two? If you release both at the same time it's more likely folks will read this post *before* the more detailed one above.